ci-operator/step-registry/gcp/deprovision/minimal-permission/gcp-deprovision-minimal-permission-commands.sh

```diff
@@ -0,0 +1,23 @@
+#!/bin/bash
+
+set -o nounset
+set -o errexit
+set -o pipefail
+
+if [ ! -f "${SHARED_DIR}/gcp_min_permissions.json" ]; then
+  echo "No 'gcp_min_permissions.json' found, abort." && exit 0
```

Comment:
If the expected file is not found, but the step is called in some workflow, I suggest to exit as failure, but not `exit 0`. So I think we can remove this if block here.